web/cobrands/fixmystreet.com/base.scss

```diff
@@ -4,16 +4,48 @@
+    text-indent: 0;
+    background: none;
+
+    svg {
+        margin-top: ((60px - 35px) / 2);
+    }
+}
+
+// Hide SVG logo by default
+.site-logo__svg {
+    display: none;
+    visibility: hidden;
 }
-.ie7 #site-logo {
-  background: url($image-sprite) -8px 0px no-repeat;
+
+// Then show it again for browsers that support inline SVGs
+// (https://codepen.io/tigt/post/inline-svg-fallback-without-javascript-take-2)
+svg|g.site-logo__svg {
+    display: inline;
+    visibility: visible;
+}
+
+.site-logo__fallback {
+    display: block;
+    width: 175px;
+    height: 0;
+    padding-top: 35px;
+    overflow: hidden;
+
+    // We don't use high-dpi-background-image() here because we don't want an SVG background.
+    background: transparent url("/cobrands/fixmystreet.com/images/site-logo-default.png") 0 0 no-repeat;
+    background-size: 175px 35px;
+    @media ($high-dpi-screen) {
+        background-image: url("/cobrands/fixmystreet.com/images/site-logo-default@2x.png");
```

Comment:
Is it worth having the 2x sprite fallbacks? Isn't this only for IE8-, do we care?

Reply:
I think only IE8 and some _very_ old versions of Android will see this fallback, so yep, we can drop the `@2x` here.